CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
+ * externalXmlSettings.xmlFileAssociations not used in getXMLSettings. See [#698](https://github.com/redhat-developer/vscode-xml/issues/698).
+ * XSD with enumeration restriction with 1000+ entries fail type assist. See [#679](https://github.com/redhat-developer/vscode-xml/issues/679).
+ * DOCTYPE entities interfering with hover annotation display of tags / attributes. See [lemminx#1212](https://github.com/eclipse/lemminx/pull/1212).
+ * Incorrect error range for `cvc-identity-constraint.4.1`. See [lemminx#1210](https://github.com/eclipse/lemminx/issues/1210).
+ * Resolve uri as system with XML catalog. See [lemminx#1199](https://github.com/eclipse/lemminx/issues/1199).
+ * Preserve invalid content while formatting. See [lemminx#1041](https://github.com/eclipse/lemminx/issues/1041).
+ * Bad element formatting is replaced with <null>. See [lemminx#1034](https://github.com/eclipse/lemminx/issues/1034).
+ * Lone quote inside tag leads to catastrophic formatting. See [lemminx#679](https://github.com/eclipse/lemminx/issues/679).
+ * XML format for bad processing instruction removes following element. See [lemminx#675](https://github.com/eclipse/lemminx/issues/675).
+ * Attribute value without a key is removed while formatting. See [lemminx#305](https://github.com/eclipse/lemminx/issues/305).
+
+### Build
+
+ * Fix native image build on Windows builders. See [#721](https://github.com/redhat-developer/vscode-xml/pull/721).
+ * Add LemMinX binary artifacts to release tab only on new changes. See [#706](https://github.com/redhat-developer/vscode-xml/pull/706).
+ * Create an entry under GH Releases for native build artifacts. See [#697](https://github.com/redhat-developer/vscode-xml/pull/697).
+ * Automatically publish latest snapshot build on Release page. See [#656](https://github.com/redhat-developer/vscode-xml/issues/656).
+ * Remove native-image GitHub action from repository. See [lemminx#1184](https://github.com/eclipse/lemminx/pull/1184).
+
+### Other
+
+ * Add DCO information to `CONTRIBUTING.md`. See [#723](https://github.com/redhat-developer/vscode-xml/issues/723).
+ * Added permission check and window error message for LemMinx binary. See [#695](https://github.com/redhat-developer/vscode-xml/pull/695).
+ * After a tag is collapsed, the content of the entire tag cannot be deleted. See [#690](https://github.com/redhat-developer/vscode-xml/issues/690).
+ * Upversion to 0.21.0. See [#688](https://github.com/redhat-developer/vscode-xml/pull/688).
+ * Update lsp4j to 0.14.0. See [lemminx#1231](https://github.com/eclipse/lemminx/pull/1231).
+ * Add test for TreeLineTracker.getLineInformation. See [lemminx#1228](https://github.com/eclipse/lemminx/pull/1228).
+ * Move to LSP4j 0.13.0. See [lemminx#1204](https://github.com/eclipse/lemminx/issues/1204).
+ * A test file contains "private non-commercial use" clause. See [lemminx#1197](https://github.com/eclipse/lemminx/issues/1197).
```

Comment:
I don't think these are worth mentioning here, since they only pertain to lemminx, and not anything that the end user will see in the vscode-xml extension

Reply:
removed